### PR TITLE
Release 3.0.0 prep

### PR DESCRIPTION
**Pull request by jspaaks**
This PR is a preparation for releasing version 3.0.0 of the Research Software Directory

Refs: #660

Supersedes: #664

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
+        - `imageCaption`
+    - renamed required properties to `project` collection
+        - `tags` to `technologies`
+- the sample data from `database/db-init` was updated to schema version 3.0.0. For notes on
+  how to migrate your own data, please refer to the
+  [data migration notes](/data-migration/2.0-to-3.0/README.md).
+- the sample data also include data harvested from esciencecenter.nl/projects such as logos,
+  related people, related organizations, descriptive text, and a hero image; furthermore the
+  project start dates and end dates were added based on data from Exact. 
+- FIXME: add commit comparison [2.0.2...3.0.0](https://github.com/research-software-directory/research-software-directory/compare/2.0.2...3.0.0)
```
Comment (jspaaks) [inline CHANGELOG.md:57]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,60 @@
-# 3.0.0 (Unreleased)
-
-See [data migration notes](/data-migration/2.0-to-3.0/README.md).
+# 3.0.0
+
+- Added project page accessible through new route `/projects/<id>` and `/projects/<slug>`
```
Comment (jspaaks) [inline CHANGELOG.md:3]:
```suggestion
commit comparison [2.0.2...3.0.0](https://github.com/research-software-directory/research-software-directory/compare/2.0.2...3.0.0)


- Added project page accessible through new route `/projects/<id>` and `/projects/<slug>`
```

File: CITATION.cff [hunk 3]
```diff
@@ -16,26 +16,35 @@ authors:
...
+    affiliation: "Netherlands eScience Center"
+    family-names: Tjong Kim Sang
+    given-names:  Erik
+    orcid: "https://orcid.org/0000-0002-8431-081X"
   -
     affiliation: "Netherlands eScience Center"
     family-names: Pawar
-    given-names: Pushpanjali
-  - 
+    given-names: Pushpanjali
```
Comment (jspaaks) [inline CITATION.cff:37]:
```suggestion
    given-names: Pushpanjali
```

File: CITATION.cff [hunk 4]
```diff
@@ -16,26 +16,35 @@ authors:
...
-    orcid: "https://orcid.org/0000-0002-6478-3003"
+    family-names: Meijer
+    given-names:  Christiaan
+    orcid: "https://orcid.org/0000-0002-5529-5761"    
   - 
     affiliation: "Netherlands eScience Center"
     family-names: Ridder
     given-names: Lars
-    orcid: "https://orcid.org/0000-0002-7635-9533"
+    orcid: "https://orcid.org/0000-0002-7635-9533"
```
Comment (jspaaks) [inline CITATION.cff:47]:
```suggestion
    orcid: "https://orcid.org/0000-0002-7635-9533"
```

**Comment by jspaaks** [comment]
we can change the order of arrays now in the admin

**Comment by jspaaks** [comment]
did we also work on more reliable harvesting with 429 errors and such?

**Comment by jspaaks** [comment]
scss in frontend project index template:
1. gutter in card set margin-top to 1em
1. finished tag for short subtitles not in the corner of the card
1. content class of card set padding-left, right top to 20px, bottom to 10px
1. thumb of card set width to 140 px, maybe remove height
1. title of page, create more space around the work Projects
1. search bar replace project singular with projects plural

**Comment by fdiblen** [comment]
all the suggestions were implemented in the last commit.

**Review by fdiblen — APPROVED** [review]
Amazing! Ready to release. :100: :horse_racing:

**Comment by jspaaks** [comment]
Yay we did it!

**Pull request merged**